renku/core/models/git.py

```diff
@@ -44,11 +44,11 @@
 
 _RE_PORT = r":(?P<port>\d+)"
 
-_RE_PATHNAME = r"(?P<pathname>(([\w\-\~\.]+)/)*?(((?P<owner>([\w\-\.]+/?)+)/)?(?P<name>[\w\-\.]+)(\.git)?)?)"
+_RE_PATHNAME = r"(?P<pathname>(([\w\-\~\.]+)/)*?(((?P<owner>([\w\-\.]+/?)+)/)?(?P<name>[\w\-\.]+)(\.git)?)?)/*"
```

Comment:
I'm wondering why you need a match for anything after the `/`? I thought this is specifically a slash at the end of the URL, i.e. by definition nothing will come after it.

Reply:
I think this is for multiple slashes at the end, which is a valid url (like http://github.com/project/repo/////)

Reply:
🤦 of course. I shouldn't review PRs on my phone...